### ISSUE TITLE
GUI to start using config files

### ISSUE THREAD

**Issue by jaimecp89** [pylhc/Beta-Beat.src#73]
Should the GUI start using configuration files instead of massive command line arguments lists?
This should work once we introduce the entrypoint in each script right @JoschD ?

**Comment by JoschD** [comment]
Will work, but you need to rewrite the GUI then to output config files... 
Advantage would be readability and easier rerun (just run the function with the path to the config file).

Imo best option. Can you vote here somehow?

**Comment by elenafol** [comment]
Sounds good, but I would see it as a minor problem (means to be done later), since we have more important things as testing of minimal branch, Andreas GetLLm, fixes in the kmod GUI and more.

**Comment by jaimecp89** [comment]
Yeah we should do it later, but I agree is the way to go. I will make this a TODO for the future.

**Comment by JoschD** [comment]
Dang, I just realized a big problem with entrypoint and config files (well, actually GUI and entrypoint):
So far you can use config-files (or json files.. are we switching to yaml? :P)  as input... but not as commandline parameter. (i.e. the argument parser does not know about them).
 - easy solution (if possible): Call python-function directly form Java (is that possible @jaimecp89 @elenafol )

 - tobeimplemented solution: put a commandline parser in front of the actual parser who checks either for "--entry_config" or argument as path (I write this so I don't forget that this should be easy to do in a try-catch-environment)

**Comment by jaimecp89** [comment]
Actually I think we discussed the --entry_config thing (in the time it was
called --config) and we agreed on it. Calling a Python function from java
is tricky if you are not using jython...
